src/Sulu/Component/Content/Metadata/Loader/XmlLegacyLoader.php

```diff
@@ -258,6 +258,7 @@ private function loadProperty($templateKey, \DOMXPath $xpath, \DOMNode $node, &$
 
         $result['mandatory'] = $this->getValueFromXPath('@mandatory', $xpath, $node, false);
         $result['multilingual'] = $this->getValueFromXPath('@multilingual', $xpath, $node, true);
+        $result['onInvalid'] = $this->getValueFromXPath('@onInvalid', $xpath, $node);
```

Comment:
I have decided for `onInvalid` instead of `on-invalid` as in the service configuration. This is because we also already have `maxOccurs`and `minOccurs` as attributes in this file.